src/front-end/typescript/lib/pages/organization/create.tsx

```diff
@@ -83,10 +83,9 @@ const view: ComponentView<State, Msg> = ({ state, dispatch }) => {
       <Row>
         <Col xs='12'>
           <OrgForm.view
-            submitHook={() => {}}
             state={state.govProfile}
             disabled={false}
-            dispatch={mapComponentDispatch(dispatch, value => adt('govProfile' as const, value))} />
+            dispatch={mapGlobalComponentDispatch(dispatch, value => adt('govProfile' as const, value))} />
```

Comment:
Successfully creating an organization should redirect to `userProfile` where `tab = 'organizations'`